doc/services/device_mgmt/smp_transport.rst

```diff
@@ -0,0 +1,197 @@
+.. _mcumgr_smp_transport_specification:
+
+SMP Transport Specification
+###########################
+
+The documents specifies information needed for implementing server and client
+side SMP transports.
+
+.. _mcumgr_smp_transport_ble:
+
+BLE (Bluetooth Low Energy)
+**************************
+
+SMP communication relays on a single GATT characteristic. An SMP command may
+be sent in the form of either GATT write command or GATT Write Without Response
+command.
+SMP responses are sent in the form of a GATT Notification, specifying the same
+characteristics as GATT Write command.
+
+BLE (Bluetooth Low Energy) Characteristics
+==========================================
+
+MCUmgr Clients need to use following BLE Characteristics, when implementing
+SMP client:
+
+- **Service UUID**: `8D53DC1D-1DB7-4CD3-868B-8A527460AA84`
+- **Characteristic UUID**: `DA2E7828-FBCE-4E01-AE9E-261174997C48`
+
+The  "SMP" GATT service consists of one **write no-rsp characteristic**
+for SMP requests: a single-byte characteristic that
+can only accepts write-without-response commands. The contents of
+each write command contains an SMP request.
+
+SMP responses are sent back in the form of unsolicited notifications
+from the same characteristic.
+
+.. _mcumgr_smp_transport_uart:
+
+UART/serial
+***********
+
+SMP protocol specification by MCUmgr subsystem of Zephyr uses basic framing
+of data to allow multiplexing of UART channel. Although no such transport
+exists in Zephyr, it is possible to implement MCUmgr client/server over
+uart transport that does not have framing at all, or uses hardware serial
+port control, or other means of framing.
+
+
+Frame fragmenting
+=================
+
+SMP protocol over serial is fragmented into MTU size frames; each
```

Comment:
It's defined as over console: https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/mgmt/mcumgr/transport/smp-console.md
Though I would be fine for updating that to have SMP over serial, either as an addition or replacing the existing over console message

Reply:
Outstanding comment